src/components/common/Modal/index.jsx

```diff
@@ -0,0 +1,22 @@
+import ReactDom from 'react-dom';
+import PropTypes from 'prop-types';
+import s from './Modal.module.scss';
+
+export const Modal = ({ closeModal, location }) => {
+  return ReactDom.createPortal(
+    <div className={s.modalOverlay} onClick={closeModal}>
+      <div
+        className={location === 'right' ? s.modalRightContent : s.modalBottomContent}
+        onClick={(e) => e.stopPropagation()}
+      >
+        <p>Modal</p>
+      </div>
+    </div>,
+    document.getElementById('portal')
+  );
+};
+
+Modal.propTypes = {
+  location: PropTypes.string,
+  closeModal: PropTypes.func
```

Comment:
no such thing as closeModal. you pass a callback that should be triggered on modal close. it means you call it onModalClose or onClose